modules/pbccs/meta.yml

```diff
@@ -0,0 +1,53 @@
+name: pbccs
+description: Pacbio ccs - Generate Higly Accurate Single-Molecule Consensus Reads
+keywords:
+  - ccs
+tools:
+  - pbccs:
+      description: pbccs - Generate Highly Accurate Single-Molecule Consensus Reads (HiFi Reads)
+      homepage: https://github.com/PacificBiosciences/pbbioconda
+      documentation: https://ccs.how/
+      tool_dev_url: https://github.com/PacificBiosciences/ccs
+      doi: ""
+      licence: ['BSD-3-clause-Clear']
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - bam:
+      type: file
+      description: Raw subreads bam
+      pattern: "*.subreads.bam"
+  - pbi:
+      type: file
+      description: Pacbio BAM Index
+      pattern: "*.pbi"
+  - chunk_num:
+      -type: integer
+      -description: BAM part to process
+  - chunk_on:
+      -type: integer
+      -description: Total number of bam parts to process
+  - rq:
```

Comment:
This doesn't have a input channel associated with it? Either add in `main.nf` or remove here